package.json

```diff
@@ -44,16 +44,16 @@
     "@types/properties-reader": "^2.1.0",
     "@types/tmp": "^0.2.0",
     "@types/yargs": "^15.0.12",
-    "@typescript-eslint/eslint-plugin": "^4.10.0",
-    "@typescript-eslint/parser": "^4.10.0",
+    "@typescript-eslint/eslint-plugin": "^4.11.0",
```

Comment:
Should these 2 dependencies be here? As we aren't using them directly and instead are depending on what `@tinymce/eslint-plugin` needs.

Reply:
```[dylan@viceversa beehive-flow]$ yarn remove @typescript-eslint/parser
yarn remove v1.22.5
[1/2] Removing module @typescript-eslint/parser...
[2/2] Regenerating lockfile and installing missing dependencies...
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
warning " > @typescript-eslint/eslint-plugin@4.11.0" has unmet peer dependency "@typescript-eslint/parser@^4.0.0".
```

Reply:
Weird that this would be a peer dependency.

Reply:
If I remove both of them, then I get an error: 

```$ yarn lint
yarn run v1.22.5
$ eslint src/**/*.ts

Oops! Something went wrong! :(

ESLint: 7.16.0

ESLint couldn't find the plugin "@typescript-eslint/eslint-plugin".

(The package "@typescript-eslint/eslint-plugin" was not found when loaded as a Node module from the directory "/home/dylan/src/tiny/beehive-flow".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install @typescript-eslint/eslint-plugin@latest --save-dev

The plugin "@typescript-eslint/eslint-plugin" was referenced from the config file in ".eslintrc.json » plugin:@tinymce/standard".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```

Reply:
As discussed on Slack, this is fine for now. I'll revisit it later to see if I can see what's going on.